WordPress/Classes/ViewRelated/Blog/SharingViewController.m

```diff
@@ -90,8 +90,7 @@ -(void)presentationControllerDidDismiss:(UIPresentationController *)presentation
 
 - (void)refreshPublicizers
 {
-    SharingService *sharingService = [[SharingService alloc] initWithManagedObjectContext:[self managedObjectContext]];
-    self.publicizeServices = [sharingService allPublicizeServices];
+    self.publicizeServices = [PublicizeService allPublicizeServicesInContext:[self managedObjectContext] error:nil];
```

Comment:
Would it be worth to pass an error pointer here, so that we can log and bail if the method throws?

I mean... if the method throws then the data the table should show won't change and this is a no-op, so there's no much waste in not handling the error.

So I guess it all boils down to whether we want to capture and log it or not. 🤔 I don't have a strong opinion about this. I'd be tempted to say "log just in case; more information is always better" but... are those logs ever looked at? I know they are sometimes for support, but not sure how much more.

Reply:
Yeah, I guess capturing the error or not depending on what we are going to do with it. If it's just logging, I would probably say it's not that valuable. Especially in this case, the error (which is Core Data failing to fetch the model objects) is not likely to happen.